src/components/LuxAlert.vue

```diff
@@ -138,8 +138,8 @@ export default {
 </script>
 
 <style lang="scss">
-@import "../assets/styles/variables.css";
-@import "../assets/styles/mixins.scss";
+@use "/src/assets/styles/variables.css" as *;
```

Comment:
```suggestion
@use "/src/assets/styles/variables";
```

I could not find good documentation for this, but it seems to work -- when you `@use` a css file, apparently you should not include the `.css` extension, and [you should not put it in the global namespace since they will be added there by default](https://stackoverflow.com/a/79445239).